### PR TITLE
Fixes #13522: Allow multiple sessions of rtf to be running at the same times

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/13522

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: rtf [hunk 1]
```diff
@@ -329,6 +331,14 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
+    while "making sure that no vagrant up are running":
+      (returncode, output) = shell("pgrep -f \"vagrant up\"", fail_exit=False)
+      if returncode == 0:
+        idle = random.randint(10, 30)
```
Comment (peckpeck) [inline rtf]: It seems a lot, what is the average vagrant up duration ?
This should be less

Comment (Fdall) [reply rtf]: I would say something around 2 minutes, 1 to boot the machine, another one to run the provisionner (which often installs Rudder).

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**